pkg/storkctl/clusterdomainupdate.go

```diff
@@ -114,15 +136,25 @@ func newDeactivateClusterDomainCommand(cmdFactory Factory, ioStreams genericclio
 					util.CheckErr(fmt.Errorf("failed to deactivate cluster domain %v: %v", clusterDomainName, err))
 					return
 				}
-				msg := fmt.Sprintf("Cluster Domain %v deactivated successfully", clusterDomainName)
+
+				msg := fmt.Sprintf("Cluster Domain deactivate operation started successfully for %v", clusterDomainName)
 				printMsg(msg, ioStreams.Out)
+
+				if waitForCompletion {
+					fmt.Printf("Deactivating..")
```

Comment:
Use `fmt.Fprintf(ioStreams.Out, <message>)`
It'll be useful for the unit test. Same for other places where you are using `fmt.Printf()`

Reply:
ok